src/components/PackTile.js

```diff
@@ -41,7 +42,9 @@ const PackTile = ({
         publicCatalogView: view,
         quickstartName: name,
       });
-      navigate(fields.slug);
+      navigate(fields.slug, {
+        state: { prevPath: location },
```

Comment:
This is why I assumed the `onRouteUpdate` change was to pass the UTM parameters from the I/O marketplace to the details of a quickstart.